hack/generate-manifest.sh

```diff
@@ -29,6 +29,7 @@ Generate a YAML manifest for Antrea using Kustomize and print it to stdout.
         --ipsec                       Generate a manifest with IPSec encryption of tunnel traffic enabled
         --all-features                Generate a manifest with all alpha features enabled
         --no-proxy                    Generate a manifest with Antrea proxy disabled
+        --endpointslice               Generate a manifest with Antrea proxy and EndpointSlice support enabled
```

Comment:
please add a check and give an error if both `--no-proxy` and `--endpointslice` are provided

Reply:
Done